Mathlib/CategoryTheory/Functor/KanExtension/Adjunction.lean

```diff
@@ -187,6 +202,7 @@ variable [HasColimitsOfShape C H]
 /-- If `G : C ⥤ H` admits a left Kan extension along a functor `L : C ⥤ D` and `H` has colimits of
 shape `C` and `D`, then the colimit of `G` is isomorphic to the colimit of a canonical functor
 `Grothendieck (CostructuredArrow.functor L) ⥤ H` induced by `L` and `G`. -/
+@[simps! (config := .lemmasOnly)]
```

Comment:
I'm surprised this is useful. Are you sure that the lemmas below are not sufficient? My expectation would be that the coherence proof in `Final.lean` becomes simpler if you only use the lemmas below and do not "unfold" `colimitIsoColimitGrothendieck` in this way, because then you essentially have to reprove one of the lemmas below, but with a lot more stuff around that can make the automation go haywire.